config/create.remote.sample-AWS.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "aws-testnet",
+  "type": "aws",
+  "nodes": 1,
```

We need at least two nodes.
```suggestion
  "nodes": 2,
```